examples/messaging/pub-sub/dotnet/Main.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Text;
+using NATS.Client;
+
+using NATS.Client;
+
+string natsURL = Environment.GetEnvironmentVariable("NATS_URL");
+if (natsURL == null)
+{
+    natsURL = "nats://127.0.0.1:4222";
+}
+
+// Create a new connection factory to create a connection.
+Options opts = ConnectionFactory.GetDefaultOptions();
+opts.Url = natsURL;
+
+// Creates a live connection to the default
+// NATS Server running locally
+ConnectionFactory cf = new ConnectionFactory();
+IConnection c = cf.CreateConnection(opts);
+
+// Here are some of the accessible properties from
+// the Msg object:
+//   Msg.Data;
+//   Msg.Reply;
+//   Msg.Subject;
+//   Msg.Header;
+//   Msg.MetaData;
```

Comment:
Should render nice as a list on the webpage: https://nats-by-example-ibrxfcrtj-connecteverything.vercel.app/examples/messaging/pub-sub/dotnet

```suggestion
//   - Msg.Data
//   - Msg.Reply
//   - Msg.Subject
//   - Msg.Header
//   - Msg.MetaData
```

Reply:
fixed